src/Tutor.Core/BuildingBlocks/FailureCode.cs

```diff
@@ -6,9 +6,12 @@ public static class FailureCode
 {
     public static readonly IError NotEnrolledInUnit = new Error("Not enrolled in unit.");
     public static readonly IError InvalidAssessmentSubmission = new Error("Invalid assessment submission.");
+    public static readonly IError InvalidRefreshToken = new Error("Refresh token is not valid.");
```

Comment:
Not a fan of this given the FailureCodes are domain-related and not application-related.

Since it's used in one method, I would probably code the `new Error("Refresh token is not valid.")` in the authentication service.